extension/js/background_page/background_page.ts

```diff
@@ -41,6 +41,7 @@ opgp.initWorker({ path: '/lib/openpgp.worker.js' });
 
   try {
     db = await ContactStore.dbOpen(); // takes 4-10 ms first time
+    await updateX509FingerprintsAndLongids(db);
```

Comment:
A small nitpick - maybe this migration `updateX509FingerprintsAndLongids` should be done after `moveContactsToEmailsAndPubkeys` is already migrated? (switch line order)

Reply:
he contact migration `moveContactsToEmailsAndPubkeys` sets correct fingerprints and longids, we only need to fix contacts migrated by the previous version